WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -104,9 +103,7 @@
     private static final int SELECT_LIBRARY_MENU_POSITION = 100;
     private static final int CLEAR_FEATURED_IMAGE_MENU_POSITION = 101;
 
-    private PostModel mPost;
-    private SiteModel mSite;
-
+    private PostSettingsListener mListener;
```

Comment:
I'm not sure about `Delegate`, as that has a certain meaning in programming which I don't think reflects the situation and can be confusing.

Also, if we did keep 'delegate' (or a similar term), it would be a delegate for the activity, not for the post settings.

One last note: I think it's fine to have a significantly different name from everything else, as it's something very different from everything else 😉.